assets/syntaxes/02_Extra/Apache.sublime-syntax

```diff
@@ -0,0 +1,221 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: Apache Conf
+file_extensions:
+  - envvars
+  - htaccess
```

Comment:
This already covers files called `htaccess`, `.htaccess` or `test.htaccess`, right? I don't think we need the additional mappings in `syntax_mapping.rs`?